lib/core/versions/latest/util/Compressor.ts

```diff
@@ -1,13 +1,17 @@
 import util = require('util');
 import zlib = require('zlib');
+import SidetreeError from '../../../../common/SidetreeError';
+import ErrorCode from '../ErrorCode';
 
 /**
  * Encapsulates functionality to compress/decompress data.
  */
 export default class Compressor {
 
+  /** The estimated ratio/multiplier of decompressed Sidetree CAS file size compared against the compressed file size. */
+  public static readonly estimatedDecomporessionMultiplier = 3;

```

Comment:
Open to suggestion on:
1. Alternative location where this constant is stored.
1. What the value should be. I chose 3 because I validated that the compression ratio is very close to 2:1 so setting to 3 means such limit would never be hit, thus resulting: a) this being more of an implementation detail rather than spec requirement, and b) equivalent check on batch writer optional/low-pri.

Reply:
1. OK with the location.
2. Is there a scenario where someone can force a node to save the max-sized decompressed file? I don't think that is the case as we validate the file contents before saving it. Calling this out as in that case we are giving a spammer more room.

Reply:
Regarding 2: You are right, we don't save decompressed file itself. This does give spammers more temporary memory to decompress into, e.g. 20MB vs 30MB, but I think it's okay because memory is released after decompression is done.